msal/src/main/java/com/microsoft/identity/client/AuthenticationActivity.java

```diff
@@ -93,6 +98,7 @@ protected void onCreate(final Bundle savedInstanceState) {
         }
 
         mUseEmbeddedWebView = data.getIntExtra(Constants.WEBVIEW_SELECTION, 0) != WebViewSelection.SYSTEM_BROWSER.getId();
+        mUseEmbeddedWebView = false;
```

Comment:
can we comment out the previous line?

Reply:
This was a workaround for enable AuthenticationActivity for testing.  I'll remove the hard-coded false value.  Still waiting for Heid's latest changes.